service/internal/auth/casbin.go

```diff
@@ -46,70 +44,49 @@ var defaultPolicy = `
 p,	role:admin,		/subject-mappings*,													*,			allow
 p,	role:admin,		/resource-mappings*,												*,			allow
 p,	role:admin,		/key-access-servers*,												*,			allow
 p,	role:admin,		/kas/v2/rewrap,						  				        *,      allow
+p,	role:admin,		/unsafe*,										            *,			allow
 p,	role:admin,		/v1/entitlements,						  				        *,      allow
 p,	role:admin,		/v1/authorization,						  				        *,      allow
 p,	role:admin,		/v1/token/authorization,						  				        *,      allow
-
+p,	role:admin,		/entityresolution/entitychain,							*,  	allow
+p,	role:admin,		/entityresolution/resolve,							*,  	allow
 
 ## Role: Standard
 ## gRPC routes
 p,	role:standard,		policy.*,																read,			allow
 p,	role:standard,		kasregistry.*,													read,			allow
-p,	role:standard,    kas.AccessService/Rewrap, 			           *,			allow
-p,  role:standard,    authorization.AuthorizationService/GetDecisions,        read, allow
-p,  role:standard,    authorization.AuthorizationService/GetDecisionsByToken, read, allow
+p,	role:standard,      kas.AccessService/Rewrap, 			           *,			allow
+p,  role:standard,      authorization.AuthorizationService/GetDecisions,        read, allow
+p,  role:standard,      authorization.AuthorizationService/GetDecisionsByToken, read, allow
+
 ## HTTP routes
 p,	role:standard,		/attributes*,														read,			allow
 p,	role:standard,		/namespaces*,														read,			allow
 p,	role:standard,		/subject-mappings*,											read,			allow
 p,	role:standard,		/resource-mappings*,										read,			allow
 p,	role:standard,		/key-access-servers*,										read,			allow
 p,	role:standard,		/kas/v2/rewrap,													write,		allow
-p,	role:standard,		/entityresolution/resolve,							write,  	allow
-p,      role:standard,          /v1/authorization,                                                              write,          allow
-p,      role:standard,          /v1/token/authorization,                                                        write,          allow
+p,  role:standard,      /v1/authorization,                                                              write,          allow
+p,  role:standard,      /v1/token/authorization,                                                        write,          allow
 
 # Public routes
 ## gRPC routes
 ## for ERS, right now we don't care about requester role, just that a valid jwt is provided when the OPA engine calls (enforced in the ERS itself, not casbin)
-p,	role:unknown,			entityresolution.EntityResolutionService.ResolveEntities,					write,		allow
 p,	role:unknown,     kas.AccessService/Rewrap, 			                                  *,	  allow
 ## HTTP routes
 ## for ERS, right now we don't care about requester role, just that a valid jwt is provided when the OPA engine calls (enforced in the ERS itself, not casbin)
-p,	role:unknown,			/entityresolution/resolve,							  write,		allow
 p,	role:unknown,		  /kas/v2/rewrap,													  *,		allow
```

Comment:
I feel like we're missing the KAS public key routes. Also, does rewrap actually support no authorization? Is that even a usecase?

Reply:
rewrap does need the token, but i guess it doesnt care about the role which i think is what this section is, not really public, casbin still requires them to provide tokens

Reply:
ah ok heres where the truly public routes are defined https://github.com/opentdf/platform/blob/88496cdb36d015fddffa2383c742350e3ed373b3/service/internal/auth/authn.go#L45-L57